frontend/src/components/AppRoutes.jsx

```diff
@@ -11,6 +12,8 @@ function AppRoutes(){
             <Route path="/" element={<BookList/>}/>
```

Comment:
@nshelokar-vf, please use 2 spaces for indentation

Reply:
Done.